VL.Data/Data/ApplicationDbContextFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace VL.Shared.Data
+{
+    public class ApplicationDbContextFactory : IDesignTimeDbContextFactory<ApplicationDbContext>
```

Comment:
You definitely do not need this

Reply:
I have spent a few hours attempting to create scaffolding without this and have had no success. Once the scaffolding has been completed, I can remove the code after the fact. The Microsoft.EntityFrameworkCore.Design package does not remove the scaffolding error in relation to access ApplicationDbContext. "This factory class creates ApplicationDbContext at design time and scaffolding runs correctly." - https://github.com/dotnet/Scaffolding/issues/1765

Reply:
I can help you with this, create a branch demonstrating your original issue that is a PULL REQUEST back into this branch